Source/g_game.c

```diff
@@ -2476,6 +2492,16 @@ void G_DeferedInitNew(skill_t skill, int episode, int map)
   d_episode = episode;
   d_map = map;
   gameaction = ga_newgame;
+
+  if (demorecording)
+  {
+    char *tmp;
+    G_CheckDemoStatus();
+    tmp = M_StringDuplicate(demoname);
+    G_RecordDemo(tmp);
+    G_BeginRecording();
```

Comment:
The demo header gets written during G_BeginRecording, which includes the rngseed. But the rngseed should be changed for each restart - I believe that happens when starting a new game. That would imply that G_BeginRecording needs to happen after the new game code runs. This is how it's set up in dsda-doom. Is it different in Woof?

Reply:
> That would imply that G_BeginRecording needs to happen after the new game code runs.

You are right, thank you. Hopefully fixed now.